### PR TITLE
feat: add bootstrap-vue intellisense to language server

### PR DESCRIPTION
**Pull request by alexsasharegan**
This an attempt to follow the Element UI team's example PR in order to add intellisense to the bootstrap-vue library. Very new to this, so all feedback welcome.

### CODE REVIEW

**Comment by HerringtonDarkholme** [comment]
@octref Any suggestion?

If we're to merge this, a common library tag provider is better than current code duplication.

**Comment by alexsasharegan** [comment]
@HerringtonDarkholme, it just looks like we need a nice way to resolve our tags and attributes sources.

File: server/src/modes/template/tagProviders/common.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ interface StandaloneAttribute {
   documentation?: string;
 }
 
-// Note: cannot items more than 10 for lexcial order
```
Comment (octref) [inline server/src/modes/template/tagProviders/common.ts]: Thanks for fixing the typo!

**Comment by octref** [comment]
@alexsasharegan Merged and thank you!

**Comment by alexsasharegan** [comment]
Thank you! I'll probably see how this all plays out and see what bootstrap-vue can do to aid in the metadata generation. Thanks for helping guide me along!

**Pull request merged**